src/main/java/backend/view/user/UserView.kt

```diff
@@ -12,7 +12,7 @@ import org.hibernate.validator.constraints.SafeHtml.WhiteListType.NONE
 import javax.validation.Valid
 import javax.validation.constraints.NotNull
 
-class UserView() {
+open class UserView() {
```

Comment:
Was bedeutet das "open"?

Reply:
Standardmäßig sind Klassen in Kotlin `final`, d.h. durch das `open` wird es möglich diese zu erschreiben und woanders um ein nur an dieser Stelle notwendiges Feld wie `newPassword` zu ergänzen.